File: app-java/src/main/java/net/mEmoZz/yts/java/ui/base/BaseInteractorImpl.java
package net.mEmoZz.yts.java.ui.base;

import net.mEmoZz.yts.java.data.network.connection.RetroConnect;
import net.mEmoZz.yts.java.data.network.webservice.BaseApi;

/**
 * Authored by Mohamed Fathy on 11 Mar, 2018.
 * Contact: muhamed.gendy@gmail.com
 */

public class BaseInteractorImpl {

  protected BaseApi api = new RetroConnect().initRetrofit();
}

File: app-kotlin/src/main/java/net/mEmoZz/yts/kotlin/ui/base/BaseInteractorImpl.kt
package net.mEmoZz.yts.kotlin.ui.base

import net.mEmoZz.yts.kotlin.data.connection.RetroConnect

/**
 * Authored by Mohamed Fathy on 11 Mar, 2018.
 * Contact: muhamed.gendy@gmail.com
 */

open class BaseInteractorImpl {

  protected var api = RetroConnect().initRetrofit()
}
